src/locales/en.json

```diff
@@ -361,7 +361,7 @@
       "instruction": "Remember to press stop when you're done. <fallback>Redo video actions</fallback>"
     },
     "camera_inactive": {
-      "message": "Camera not working?",
+      "message": "Camera not working",
```

Comment:
I believe this message is displayed when the user is not taking a photo for more than n seconds. I think that the new copy might be misleading

Reply:
I'll revert the string change for now, even though minor, since we have the Lokalise strings work ongoing & this was more to fix the bisu script breaking.